src/plugins/data/common/search/aggs/types.ts

```diff
@@ -170,3 +202,41 @@ export interface AggParamsMapping {
   [METRIC_TYPES.SERIAL_DIFF]: AggParamsSerialDiff;
   [METRIC_TYPES.TOP_HITS]: AggParamsTopHit;
 }
+
+/**
+ * A global list of the expression function definitions for each agg type function.
+ */
+export interface AggFunctionsMapping {
```

Comment:
This mapping isn't strictly necessary since you could import each type individually, but I found it much easier to work with when a mapping was available, since there are so many agg types.